lang/python/python-package-install.sh

```diff
@@ -1,11 +1,6 @@
 #!/bin/sh
 set -e
 
-[ -z "$SOURCE_DATE_EPOCH" ] || {
```

Comment:
phew... big change
looks good at first;
[if you want, or feel like doing] one way to test this, is actually using `SOURCE_DATE_EPOCH`, then build the packages with the old code, then with this change;
if binaries are identical, it should be good;

i don't remember if Py3 reproduce-able builds were 100%, but even 70/80% is good enough to validate tihs;

Reply:
It's still using `SOURCE_DATE_EPOCH` [here](https://github.com/openwrt/packages/pull/11872/commits/3cdca38dce01553766b032d8bbfce08a35d6555e#diff-b8485961a6fc60b60a8f79736845c998R102).

I've looked through the bpo issues; setting `PYTHONHASHSEED` still seems to be necessary for reproducibility. Personally I would prefer not setting it, as the value shouldn't be hard for an attacker to find and I'm not sure what effect it has for target Python (whether it weakens security or not), but I'm ok to keep setting it for now.